### PR TITLE
Readme Review

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <maryjinglewski@honeycomb.io>

## Which problem is this PR solving?
N/A

## Short description of the changes
This PR consists of copyedits for the project README.

## How to verify that this has the expected result
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,18 @@
...
 ![AWS Integrations architecture](docs/overview.png?raw=true)
 
-Note: [Terraform modules](https://github.com/honeycombio/terraform-aws-integrations) are also available that support AWS to Honeycomb Integrations.
-
 ## Usage
 
-A [quick start template](README.md#quick-start) is available to offer an easy path to integrating your AWS environments with Honeycomb.
-It makes use of the of the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
+A [quick start template](README.md#quick-start) offers a streamlined path to integrating your AWS environments with Honeycomb.
+The template uses the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
```
Comment (mjingle) [inline README.md:16]: Question for Team - I'm not sure if "template" is the preferred subject of this sentence, but ideally would like to fix this for clarify.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,18 @@
...
 
 The quick start template may be suitable for many production purposes, but we encourage you to use per-integration templates in a way which suits your AWS environment.
```
Comment (mjingle) [inline README.md:18]:
```suggestion
The quick start template may be suitable for many production purposes, but we encourage you to use per-integration templates in a way that suits your AWS environment.
```

File: README.md [hunk 3]
```diff
@@ -38,22 +37,22 @@ This 'Quick Start' template allows the configuration of multiple integrations fr
 
 ### Required Inputs
 
-- `HoneycombAPIKey`: : Your Honeycomb Team's API Key.
+* `HoneycombAPIKey`: : Your Honeycomb Team's API Key.
```
Comment (mjingle) [inline README.md:40]: Sorry, I think my linter got carried away with the type of bullet points.
```suggestion
- `HoneycombAPIKey`: : Your Honeycomb Team's API Key.
```

File: README.md [hunk 4]
```diff
@@ -38,22 +37,22 @@ This 'Quick Start' template allows the configuration of multiple integrations fr
...
 ### Required Inputs
 
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
-- `LogGroupName`: A CloudWatch Log Group name. Additional Log Groups can be added with the `LogGroupNameX` parameters.
-- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
+* `LogGroupName`: A CloudWatch Log Group name. Additional Log Groups can be added with the `LogGroupNameX` parameters.
+* `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```
Comment (mjingle) [inline README.md:55]: Also, what is ARN? It's not defined anywhere in this page.

Comment (mjingle) [inline README.md:55]: Also what is ARN? It's not defined on this page anywhere.

File: README.md [hunk 5]
```diff
@@ -63,8 +62,8 @@ This template supports integrating with all metrics flowing to Cloudwatch Metric
 
 ### Required Inputs
 
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```
Comment (mjingle) [inline README.md:66]:
```suggestion
- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```

File: README.md [hunk 6]
```diff
@@ -74,10 +73,10 @@ This template creates a Kinesis Firehose Stream that streams data received to Ho
...
 ### Required Inputs
 
-- `Name`: A Name for this Kinesis Firehose. Must be unique in this Region.
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
-- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
+* `Name`: A Name for this Kinesis Firehose. Must be unique in this Region.
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
+* `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```
Comment (mjingle) [inline README.md:79]:
```suggestion
- `Name`: A Name for this Kinesis Firehose. Must be unique in this Region.
- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```

File: README.md [hunk 7]
```diff
@@ -87,23 +86,21 @@ This template streams RDS logs from Cloudwatch to a Kinesis Firehose that includ
...
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
-- `DBEngineType`: The Engine type of your RDS database. One of `aurora-mysql`, `aurora-postgresql` `mariadb`, `sqlserver`,`mysql`, `oracle`, or `postgresql`.
-- `LogGroupName`: A CloudWatch Log Group name for RDS logs. Additional Log Groups can be added with the `LogGroupNameX` parameters.
-- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
+* `DBEngineType`: The Engine type of your RDS database. One of `aurora-mysql`, `aurora-postgresql` `mariadb`, `sqlserver`,`mysql`, `oracle`, or `postgresql`.
+* `LogGroupName`: A CloudWatch Log Group name for RDS logs. Additional Log Groups can be added with the `LogGroupNameX` parameters.
+* `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```
Comment (mjingle) [inline README.md:93]:
```suggestion
- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
- `DBEngineType`: The Engine type of your RDS database. One of `aurora-mysql`, `aurora-postgresql` `mariadb`, `sqlserver`,`mysql`, `oracle`, or `postgresql`.
- `LogGroupName`: A CloudWatch Log Group name for RDS logs. Additional Log Groups can be added with the `LogGroupNameX` parameters.
- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```

File: README.md [hunk 8]
```diff
@@ -87,23 +86,21 @@ This template streams RDS logs from Cloudwatch to a Kinesis Firehose that includ
...
 
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `HoneycombDataset`: The target Honeycomb dataset for to publish to.
-- `ParserType`: The type of log file to parse. One of `alb`, `elb`, `cloudfront`, `keyval`, `json`, `s3-access`, or `vpc-flow`.
-- `S3BucketArn`: The ARN of the S3 Bucket storing the logs.
-
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `HoneycombDataset`: The target Honeycomb dataset for to publish to.
+* `ParserType`: The type of log file to parse. Choose one of `alb`, `elb`, `cloudfront`, `keyval`, `json`, `s3-access`, or `vpc-flow`.
+* `S3BucketArn`: The ARN of the S3 Bucket storing the logs.
```
Comment (mjingle) [inline README.md:106]:
```suggestion
- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
- `HoneycombDataset`: The target Honeycomb dataset for to publish to.
- `ParserType`: The type of log file to parse. Choose one of `alb`, `elb`, `cloudfront`, `keyval`, `json`, `s3-access`, or `vpc-flow`.
- `S3BucketArn`: The ARN of the S3 Bucket storing the logs.
```

File: README.md (hunk 1 above)
Comment (mjingle) [reply README.md:16]: Yep. The question is "which template" here :) Thanks!
```suggestion
The quick start template uses the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
```

File: README.md (hunk 4 above)
Comment (jharley) [reply README.md]: Hmm.. tricky. We don't define S3 or CloudWatch either and all are AWS specific jargon.

The AWS definition: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html

Comment (mjingle) [reply README.md]: Well, S3 and CloudWatch are more mainstream as product name while ARN as a term is new to me.

...we _can_ define those terms, too, at least once though...😈

File: README.md [hunk 9]
```diff
@@ -2,21 +2,20 @@
...
 ![AWS Integrations architecture](docs/overview.png?raw=true)
 
-Note: [Terraform modules](https://github.com/honeycombio/terraform-aws-integrations) are also available that support AWS to Honeycomb Integrations.
-
 ## Usage
 
-A [quick start template](README.md#quick-start) is available to offer an easy path to integrating your AWS environments with Honeycomb.
-It makes use of the of the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
+A [quick start template](README.md#quick-start) offers a streamlined path to integrating your AWS environments with Honeycomb.
+The quick start template uses all of the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
```
Comment (mjingle) [inline README.md:16]:
```suggestion
The quick start template uses all of the per-integration templates below to offer the configuration of many integrations in a single CloudFormation stack.
```

File: README.md [hunk 10]
```diff
@@ -99,11 +98,9 @@ This template supports sending logs flowing to a S3 bucket to Honeycomb.
 
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https://honeycomb-builds.s3.amazonaws.com/cloudformation-templates/latest/s3-logfile.yml&stackName=Honeycomb-S3-Logs" target="_blank"><img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png" alt="Launch Stack" /></a>
 
-
 ### Required Inputs
```
Comment (mjingle) [inline README.md:101]: @jharley When would these required inputs be encountered? In an AWS form after clicking the button?

Comment (jharley) [reply README.md:101]: They're displayed in the UI (along with a lengthy list of optional parameters) or would need to be provided via the CLI or API (CloudFormation supports three avenues)

Comment (mjingle) [reply README.md:101]: Thank you! So an applicable description to add above each bulleted list could be...?
> Enter a value for each required input in the UI, or if using the CLI or API, ensure to include each required input and its value.

File: README.md (hunk 4 above)
Comment (mjayaram) [reply README.md]: hmm I feel like lets wait and iterate on how much we need to explain AWS domain? I feel like knowing what it stands for might not help someone who does not know what it is and how to get it - they might need to go look it up in AWS docs anyway?

File: README.md (hunk 10 above)
Comment (jharley) [reply README.md:101]: Yep, that works!

Comment (mjingle) [reply README.md:111]: Added in the most recent commit. Thanks!

File: README.md (hunk 4 above)
Comment (mjingle) [reply README.md:59]: Okay. Will hold off then.

**Comment by mjingle** [comment]
@jharley @mjayaram This PR should be good to merge when y'all are ready.

**Review by jharley — APPROVED** [review]

**Pull request merged**